frontend/app/helpers/spree/frontend_helper.rb

```diff
@@ -181,6 +181,17 @@ def image_source_set(name)
       set.join(', ')
     end
 
+    def og_image_pdp(product)
```

Comment:
@devilcoders this tag is already added via https://github.com/spree/spree/pull/9809